pkg/restapi/operation/openapi.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package operation
+
+import "github.com/trustbloc/edv/pkg/restapi/models"
+
+// genericError model
+//
+// swagger:response genericError
+type genericError struct { // nolint: unused,deadcode
+	// in: body
+	ErrMsg string
+}
+
+// createVaultReq model
+//
+// swagger:parameters createVaultReq
+type createVaultReq struct { // nolint: unused,deadcode
+	// in: body
+	NewVaultRequest models.DataVaultConfiguration
+}
+
+// createVaultRes model
+//
+// swagger:response createVaultRes
+type createVaultRes struct { // nolint: unused,deadcode
+	// For some reason this doesn't display in the Swagger response UI currently...
```

Comment:
dirty laundry comments in open api specification, remove/move it

Reply:
just specify it will be part of response header

Reply:
Done.